tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -412,6 +412,11 @@ func getExec(artifact string, kind string, isDocker bool, mainEntry string) (*wh
         code, err = readFile(artifact)
         exec.Code = &code
 
+        if (ext == ".jar") {
```

Comment:
I think we might be able to Base64 encode here for .jar and .zip files. And remove the encoding for .zip files that is at the end of the function.

```
    if ext == “.jar” || ext == “.zip”  {
        code = base64.StdEncoding.EncodeToString([]byte(code))
        exec.Code = &code
    }

```